test/services/validation/CSOPTestData/ERSValidationCSOPRCLTestData.scala

```diff
@@ -63,7 +63,6 @@ trait ERSValidationCSOPRCLTestData {
         "Return Enter an employer PAYE reference. For example '123/AB456'. When the submitted text is empty.",
         //Column I
         "When payeOperatedApplied is Yes or No, no validation error should be raised",
-        "Return This entry must be 'yes' or 'no'. When an empty string is given for payeOperatedApplied.",
```

Comment:
Dont delete this test case? Surly it still applies, it may just need updating to work correctly?

Reply:
I think you can just update the expected result to `None`, this case shouldn't throw an error anymore because its no longer a mandatory field (same is true for the checking PR as well)?

Reply:
added the empty checks and made it expected results to none